kubernetes/monitoring/services/mongo/monit-mongo.yaml

```diff
@@ -0,0 +1,164 @@
+# [ConfigMap] [Service] [StatefulSet]
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: monit-mongo-configmap
+  namespace: monit-mongo
+data:
+  mongo.conf: |
+    storage:
+      dbPath: /data/db
+  ensure-users.js: |
```

Comment:
it is better to put actual files into repository area and package them with the image. This will make file available and easy to debug. Here you make it as part of k8s manifest, and even though it will be created someone cannot use it until stuff deployed to k8s, while if it is part of the image, I can run docker run and debug everything within container without needs to have k8s infrastructure.

Reply:
Thanks for the suggestion. Yes, today I dealt with a similar debugging JS case. I created a Dockerfile `monit-mongo`, built the image and pushed to cern registry. Now, JS and conf file are moved from ConfigMap to Docker image.